accessibility-checker-engine/help-v4/en-US/aria_activedescendant_tabindex_valid.html

```diff
@@ -85,7 +85,7 @@ <h3 id="ruleMessage"></h3>
 
 * [IBM 2.1.1 Keyboard](https://www.ibm.com/able/requirements/requirements/#2_1_1)  
 * [ARIA practices - Managing focus with aria-activedescendant](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_focus_activedescendant)
-* [HTML - the tabindex attribute](https://html.spec.whatwg.org/multipage/interaction.html#the-tabindex-attribute)
+* [HTML - Tabindex attribute](https://html.spec.whatwg.org/multipage/interaction.html#the-tabindex-attribute)
```

Comment:
Attributes are generally all lower case
```suggestion
* [HTML - tabindex attribute](https://html.spec.whatwg.org/multipage/interaction.html#the-tabindex-attribute)
```